psst-core/src/player/queue.rs

```diff
@@ -18,7 +18,9 @@ impl Default for QueueBehavior {
 
 pub struct Queue {
     items: Vec<PlaybackItem>,
+    added_items: Vec<PlaybackItem>,
```

Comment:
Maybe there is a better name for this var? I'm not sure what it would be but `added_items` vs `items` are not clearly distinct in their usage from the names alone.

Reply:
Do you think that user_items would be clearer?

Reply:
I will change it to user_added_items

Reply:
User items works as well